server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts

```diff
@@ -155,20 +155,20 @@ export class AlfredResourcesFactory implements core.IResourcesFactory<AlfredReso
         const redisClientForJwtCache = new Redis(redisOptions2);
         const redisJwtCache = new services.RedisCache(redisClientForJwtCache);
 
+        const bufferMaxEntries = config.get("mongo:bufferMaxEntries") as number | undefined;
         // Database connection for global db if enabled
         let globalDbMongoManager;
         const globalDbEnabled = config.get("mongo:globalDbEnabled") as boolean;
         if (globalDbEnabled) {
             const globalDbMongoUrl = config.get("mongo:globalDbEndpoint") as string;
-            const globalDbMongoFactory = new services.MongoDbFactory(globalDbMongoUrl);
-            globalDbMongoManager = new core.MongoManager(globalDbMongoFactory);
+            const globalDbMongoFactory = new services.MongoDbFactory(globalDbMongoUrl, bufferMaxEntries);
+            globalDbMongoManager = new core.MongoManager(globalDbMongoFactory, false);
```

Comment:
This is the shouldReconnect flag? Why do we want to make it false?

Reply:
This is false everywhere in prod. It is only set to true in tests. What this does is create a brand new connection pool, which would be an expensive way to reconnect in prod. Instead of that, we have autoreconnect set to true in the MongoClientOptions which means the Mongo driver automatically tries to reconnect existing connections within the same connection pool. This is how CosmosDB also recommends doing it.